src/platform_impl/linux/wayland/event_loop/mod.rs

```diff
@@ -230,7 +230,7 @@ impl<T: 'static> EventLoop<T> {
             // Reset the internal state for the loop as we start running to
             // ensure consistent behaviour in case the loop runs and exits more
             // than once.
-            self.set_control_flow(ControlFlow::Poll);
```

Comment:
You can keep the `Poll` here, I think motivation is to do the first loop iteration non-blocking, because if you see the code, it won't get reset anymore? We just need to change the user default, but if the code really hardcodes the poll it's likely only a one time action.